typescript/src/start-script.ts

```diff
@@ -108,10 +108,9 @@ protocCommand.push(...protoDirsArg);
 protocCommand.push(...protoFiles);
 protocCommand.push(`-I${commonProtoPath}`);
 try {
-  execFileSync(`protoc`, protocCommand, { stdio: 'inherit' });
+  execFileSync(`protoc`, protocCommand, {stdio: 'inherit'});
 } catch (err) {
-  console.error(err.toString());
-  process.exit(1);
+  throw new Error(err.toString());
```

Comment:
No real point in catching an error just to rethrow it I'd imagine

Reply:
remove this try..catch block. Thx!